### DISCUSSION TITLE
关于上海杉达学院祝欣蓉老师的连接

### DISCUSSION THREAD

**Discussion by will-ww** [X-lab2017/open-wonderland#378] [1️⃣ 新鲜事]
去年下半年，在开放原子开发者大会等场合，认识了上海杉达学院的祝欣蓉老师，一位非常活跃的高校开源推动者。

依托所在的学校，做了非常多的推动开源的事情，可以参见：
- [相信开发的力量](https://consumer.huawei.com/cn/press/news/2022/the-path-of-a-female-teacher-to-introduce-harmonyos-courses-to-universities/)
- [《开源技术创新应用》微专业](https://www.openatom.org/journalism/detail/SXMRHwizpGHM)
- [ 鸿蒙原生应用生态](https://shanghai.eol.cn/shanghainews/202402/t20240202_2557779.shtml)

作为同类高校的标杆，共建符合其特点的开源教育体系，让更多的师生加入到开源共创中来，令人期待。可以重点开展的事宜包括：

**1、共建标杆类开源教育课程**

依托上海杉达学院的特点，选取“开源教育全景图”中适当的模块，和已有的课程或微专业进行结合，推动高校开源教育的落地。

**2、在上海开展更多的开源教育活动**

依托徐汇滨江、临港等地，开源开源教育相关活动，传播已有的开源教育经验，吸引并连接更多的同类老师。

**3、在上海杉达学院开展基于 OpenRank 的开源贡献度评估与激励活动**

依托开源平台与项目，通过将 OpenRank 开源贡献度同学分选修、奖学金评定、保研加分等政策进行挂钩，推动高校师生积极参与开源活动。

更多事宜，欢迎随时提出~

**Comment by yifanlinshui** [comment]
2024年3月28日，终于在一年的准备下，在我们学校成立了上海开源创新双语头马俱乐部，Toastmasters International（TI/国际演讲会）是一个非盈利的教育组织，致力于提高会员的[演讲技巧，该组织在全球范围内已遍布142个国家，拥有15500余个演讲[俱乐部，俱乐部会员数量超过33万余人。这也应该是上海第一家高校头马俱乐部，在建立这个俱乐部之前，我是有担心的，这算不算有点不顾正业？另一方面就是，因为这个也是需要费用的，目前第一期的会员是由我的项目经费支持的，后续的延续性的问题。不过我还是很赞同这个俱乐部的理念，和开源的理念很像，希望可以越来越好，我是希望通过另一种形式，让更多的同学去了解开源，而不仅仅是技术的学习，知识的传输。

- 俱乐部的使命：提供互助互益的学习体验，使会员提高沟通和领导能力，最终达到提高自信，促进个人成长的目标

- 头马核心价值观

诚信Integrity
尊重Respect
服务他人Service
追求卓越Excellence

**Comment by yifanlinshui** [comment]
各位老师，今天我又遇到一个困惑的问题，有个学生用新技术做的项目，因为技术比较新，花了挺多时间，但实际做出来的效果确实没有成熟的技术做的漂亮或者做得好，答辩的时候得到质疑，说工作量不够，或者说没有创新，批的非常厉害，这个项目就是用鸿蒙做的，软件硬件云都加上去了，但是还是被答辩老师说这种项目就是一个课程设计，我也知道学生挺委屈的，我自己也存在了困惑，我们是不是让学生稳稳的走走大框架的传统毕业设计就可以了，不要选这种吃力不讨好的项目，确实内心有些矛盾。

**Reply by will-ww** [reply]
完全理解这种情况！这个就需要对我们高校的老师进行“教育”了，很多老师也还停留在表面，也不懂整个产业和国家战略的发展。

而且这一点就更加需要依托类似基金会的组织来进行托底与宣传了，我们可以在这块提供些支持和背书，告诉目前还不太明白的老师们，项目的意义所在。

**Comment by yifanlinshui** [comment]
今天晚上又遇到一个问题：目前开源的项目越来越多，经常会有一些项目的宣讲会之类的，如何帮助刚入开源的同学们挑选适合他们的开源项目去关注，或者如何引导学生去寻找到自己真正感兴趣的开源项目，还是说，作为老师，指定一些开源项目，让同学们去了解和关注，如何保证学生的持续性，从了解——关注——贡献的过程。

**Reply by will-ww** [reply]
这个确实也是个好问题，不是所有的项目都适合学生去参与的，这个需要在教育侧进行设计，然后在分一些类别（前端、后端、数据库、人工智能、Pyton、Go 等），这样方便老师和学生进行参与。

这件事情，我们联合基金会，到时可以一起来推动下~
